Silicon/PantherlakePkg/Library/BaseSerialIoI2cLib/BaseSerialIoI2cLib.inf

```diff
@@ -0,0 +1,38 @@
+/** @file
+
+  Copyright (c) 2025 - 2024, Intel Corporation. All rights reserved.<BR>
```

Comment:
@ssbyna the year usually should be 2024 - 2025, or you can leave 2025 only. same for other files in this lib. fyi